src/common/sdk/web/_openlogin-options.mdx

```diff
@@ -15,7 +15,7 @@ import Tabs from "@theme/Tabs";
 
 | Variable                | Type                                  | Description                                                                                                                                                                                                                                                                                                                                               | Required | Default Value |
 | ----------------------- | ------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- | ------------- |
-| `clientId`              | `string`                              | Client ID for web3auth. Obtain your `clientId` from the [Web3Auth Developer Dashboard](https://dashboard.web3auth.io/).                                                                                                                                                                                                                                   | Yes      | N/A           |
+| `clientId?`             | `string`                              | Client ID for web3auth. Obtain your `clientId` from the [Web3Auth Developer Dashboard](https://dashboard.web3auth.io/).                                                                                                                                                                                                                                   | No       | N/A           |
```

Comment:
can you add same note here, that is it required here if not being passed in web3auth constructor.

Reply:
done